src/object-number.scss

```diff
@@ -0,0 +1,63 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-number+(--success | --warning | --error)
+*/
+$block: #{$fd-namespace}-object-number;
+
+$color-states: (
+  "positive": ("color": var(--sapPositiveTextColor)),
+  "critical": ("color": var(--sapCriticalTextColor)),
+  "negative": ("color": var(--sapNegativeTextColor)),
+  "informative": ("color": var(--sapInformativeTextColor))
+);
+
+.#{$block} {
+  // TEXT VARIABLES
+  $fd-object-number-font-family: var(--sapFontFamily) !default;
+  $fd-object-number-text-font-size: var(--sapFontMediumSize) !default;
```

Comment:
These variable does not exists in sap-theming. Please check `css_variables.css` file from `node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables.css`. If you need medium size you can use `--sapFontSize` corresponding to 0.875rem